testsuite/forge/src/backend/local/mod.rs

```diff
@@ -134,11 +137,25 @@ impl LocalFactory {
             guard,
         )?;
 
+        // Launch the swarm
         swarm
             .launch()
             .await
             .with_context(|| format!("Swarm logs can be found here: {}", swarm.logs_location()))?;
 
+        // Add and launch the fullnodes
+        let validator_peer_ids = swarm.validators().map(|v| v.peer_id()).collect::<Vec<_>>();
+        for validator_peer_id in validator_peer_ids.iter().take(number_of_fullnodes) {
+            let _ = swarm
+                .add_validator_fullnode(
+                    version,
+                    NodeConfig::default_for_validator_full_node(),
+                    *validator_peer_id,
+                )
+                .unwrap();
+        }
+        swarm.wait_all_alive(Duration::from_secs(60)).await?;
```

Comment:
Should this be configurable?

Reply:
`swarm.launch` has this hard coded to `60` secs. I guess someone could come through and make them all configurable, but that won't be me in this PR 😄